templates/graphdb-node.yaml

```diff
@@ -120,6 +132,7 @@ spec:
               mountPath: /opt/graphdb/home
             - name: graphdb-license
               mountPath: /tmp/license/
+          securityContext: {{ $.Values.graphdb.node.containerSecurityContext | toYaml | nindent 12 }}
```

Comment:
I strongly advise against sharing configuration options between graphdb and the busybox image.

Reply:
I strongly agree with said advise. Miss on my part.